Sources/PointFree/Account/Cancel.swift

```diff
@@ -207,44 +206,45 @@ private func sendCancelEmail(to owner: User, for subscription: Stripe.Subscripti
     return sendEmail(
       to: [owner.email],
       subject: "Your subscription has been canceled",
-      content: inj2(cancelEmailView.view((owner, subscription)))
+      content: inj2(cancelEmailView((owner, subscription)))
     )
 }
 
-let cancelEmailView = simpleEmailLayout(cancelEmailBodyView)
-  .contramap { owner, subscription in
-    SimpleEmailLayoutData(
-      user: nil,
-      newsletter: nil,
-      title: "Your subscription has been canceled",
-      preheader: """
-      Your \(subscription.plan.nickname) subscription has been canceled and will remain active through
-      \(dateFormatter.string(from: subscription.currentPeriodEnd)).
-      """,
-      template: .default,
-      data: (owner, subscription)
-    )
-}
-
-private let cancelEmailBodyView = View<(User, Stripe.Subscription)> { user, subscription in
-  emailTable([style(contentTableStyles)], [
-    tr([
-      td([valign(.top)], [
-        div([`class`([Class.padding([.mobile: [.all: 2]])])], [
-          h3([`class`([Class.pf.type.responsiveTitle3])], ["Subscription canceled"]),
-          p([`class`([Class.padding([.mobile: [.topBottom: 2]])])], [
-            "Your ",
-            strong([.text(subscription.plan.nickname)]),
-            " subscription has been canceled and will remain active through ",
-            .text(dateFormatter.string(from: subscription.currentPeriodEnd)),
-            ". If you change your mind before then, you can reactivate from ",
-            a([href(url(to: .account(.index)))], ["your account page"]),
-            "."
+let cancelEmailView = { owner, subscription in
+  SimpleEmailLayoutData(
+    user: nil,
+    newsletter: nil,
+    title: "Your subscription has been canceled",
+    preheader: """
+    Your \(subscription.plan.nickname) subscription has been canceled and will remain active through
+    \(dateFormatter.string(from: subscription.currentPeriodEnd)).
+    """,
+    template: .default,
+    data: (owner, subscription)
+  )
+  } >>> simpleEmailLayout(cancelEmailBodyView)
```

Comment:
super nitty, but what do you think about doing `simpleEmailLayout(cancelEmailBodyView) <<<` at the beginning instead? kinda of nice to have the layout at the top

Reply:
Can do. Just was `>>>`ing everything.